resources/lib/bossanova808/constants.py

```
@@ -18,10 +18,11 @@
 TRANSLATE = LANGUAGE = ADDON.getLocalizedString
 LOG_PATH = xbmcvfs.translatePath('special://logpath')
 KODI_VERSION = xbmc.getInfoLabel('System.BuildVersion')
+# Fallback: try System.BuildVersion (e.g. "21.0.0") then default to 21 (Omega)
 try:
-    KODI_MAJOR_VERSION = int(xbmc.getInfoLabel('System.BuildVersionCode').split('.')[0])
-except (ValueError, IndexError):
-    KODI_MAJOR_VERSION = 21  # fall back to the current major version by default
+    KODI_MAJOR_VERSION = int(xbmc.getInfoLabel('System.BuildVersion').split('.')[0])
+except Exception:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)